app/controllers/authentication_controller.rb

```diff
@@ -0,0 +1,15 @@
+class AuthenticationController < ApplicationController
+  skip_before_action :authorize_request, only: :authenticate
+
+  def authenticate
```

Comment:
Let's keep our controller action names RESTful (Index, Show, Create, Update, Destroy)

Reply:
Kindly address this one :) Let's keep our controller action names RESTful